vendor/README.md

```diff
@@ -63,6 +63,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **CLIPS:** [psicomante/CLIPS-sublime](https://github.com/psicomante/CLIPS-sublime)
 - **CMake:** [textmate/cmake.tmbundle](https://github.com/textmate/cmake.tmbundle)
 - **COBOL:** [bitbucket:bitlang/sublime_cobol](https://bitbucket.org/bitlang/sublime_cobol)
+- **CODEOWNERS:** [bmalehorn/codeowners-language](https://github.com/bmalehorn/codeowners-language)
```

Comment:
I hacked together a `CODEOWNERS` grammar in Alhadis/language-etc@fc2856e, which yields decidedly clearer output:
* [`Alhadis/language-etc`](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fgithub.com%2FAlhadis%2Flanguage-etc%2Fblob%2Fmaster%2Fgrammars%2Fcodeowners.cson&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2FAlhadis%2Flanguage-etc%2Fmaster%2Fsamples%2FCODEOWNERS&code=)
* [`bmalehorn/codeowners-language`](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fgithub.com%2Fbmalehorn%2Fcodeowners-language%2Fblob%2Fmaster%2Fsyntaxes%2Fcodeowners.tmLanguage.json&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2FAlhadis%2Flanguage-etc%2Fmaster%2Fsamples%2FCODEOWNERS&code=)

Its highlighting is also consistent with that of `.gitignore` files, which it's modelled off of.